README.md

```diff
@@ -108,66 +122,72 @@ conda activate py37deezy
+* Type of normalization and preprocessing that is to be applied to the input string, and tokenization mode (char, ngram, word).
+* Neural network architecture (RNN, GRU, or LSTM) and its hyperparameters (number of layers and directions in the recurrent units, the dimensionality of the hidden layer, learning rate, number of epochs, batch size, early stopping, dropout probability), pooling mode and layers to freeze during fine-tuning.
+* Proportion of data used for training, validation and test.
 
-These three files can be downloaded directly from `inputs` and `dataset` directories on [DeezyMatch repo](https://github.com/Living-with-machines/DeezyMatch).
+See the sample [input file](https://github.com/Living-with-machines/DeezyMatch/blob/master/inputs/input_dfm.yaml) for a complete list of the DeezyMatch options that can be configured from the input file.
 
-**Note on vocabulary:** `characters_v001.vocab` combines all characters from the different datasets we have used in our experiments 
-(See [DeezyMatch's paper](https://www.aclweb.org/anthology/2020.emnlp-demos.9/) and 
-[this paper](https://arxiv.org/abs/2009.08114) for a detailed description of the datasets). 
-It consists of 7,540 characters from multiple alphabets, containing special characters.
+### The vocabulary file
+The vocabulary file (`./inputs/characters_v001.vocab`) file combines all characters from the different datasets we have used in our experiments (see [DeezyMatch's paper](https://www.aclweb.org/anthology/2020.emnlp-demos.9/) and [this paper](https://arxiv.org/abs/2009.08114) for a detailed description of the datasets). It consists of 7,540 characters from multiple alphabets, containing special characters. You will only need to change the vocabulary file in certain fine-tuning settings.
 
-`dataset-string-similarity_test.txt` contains 9995 example rows. The original dataset can be found here: https://github.com/ruipds/Toponym-Matching.
+### The datasets
 
+We provide the following minimal sample datasets to showcase the functionality of DeezyMatch. Please note that these are very small files that have been provided just for illustration purposes.
 
-## Run DeezyMatch as a Python module or via command line
+* **String matching datasets:** The `dataset-string-matching_xxx.txt` files are small subsets from a larger [toponym matching dataset](https://github.com/ruipds/Toponym-Matching). We provide:
+  * `dataset-string-matching_train.txt`: data used for training a DeezyMatch model from scratch [5000 string pairs].
+  * `dataset-string-matching_finetune.txt`: data used for fine-tuning an existing DeezyMatch model (this is an optional step) [2500 string pairs].
+  * `dataset-string-matching_finetune.txt`: data used for assessing the performance of the DeezyMatch model (this is an optional step, as the training step already produces an intrinsic evaluation) [2495 string pairs].
```

Comment:
Should this be `_test.txt`?

Reply:
Ah, yes, it should be test, sorry!

Reply:
Great, updated here: https://github.com/Living-with-machines/DeezyMatch/pull/116/commits/475c729aa5af142e3899f62c6918c8b732ee2662